src/donejs-carousel-plugin-test.js

```diff
@@ -1026,4 +1030,167 @@ describe('tucows-donejs-carousel', () => {
 	describe('clearAutoPlay()', () => {
 		// NOTE: We are not currently testing DOM manipulation
 	});
+
+	describe('Demo Page', () => {
+		let componentElements;
+		let testArea = document.getElementById('test-area');
+
+		/**
+		 * @function beforeSetup
+		 * @description iterate through slideArray and append all the carousels
+		 * to the testArea DOM
+		 */
+		const beforeSetup = (done) => {
+			for (let i = 0; i < slideArray.length; i++) {
+				let map = new DefineMap(slideArray[i]);
```

Comment:
@kevcom : is it necessary to wrap with `DefineMap()`?

Reply:
yes it is necessary because the `activeSlideIndex` property is an observable that needs to be bound. I tried it without `DefineMap` and it didn't work